packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailResetPasswordTwoFactor/en.ftl

```diff
@@ -0,0 +1,6 @@
+automated-email-reset-pwd-two-factor = This is an automated email; if you did not authorize this action, then <a data-l10n-name="resetLink">please reset your password</a> and <a data-l10n-name="twoFactorSettingsLink">reset your two-step authentication</a>.
+  For more information, please visit <a data-l10n-name="supportLink">{ -brand-mozilla } Support</a>.
+# Followed by link to https://accounts.firefox.com/reset_password
+automated-email-reset-pwd-plaintext-v3 = If you did not authorize this action, please reset your password now at:
+# Followed by link to https://accounts.firefox.com/settings#two-step-authentication
+automated-email-reset-two-factor-plaintext = and reset your two-step authentication at:
```

Comment:
I'm not sure if this is missing something, if not, can you make this a complete sentence on its own to reduce the complexity for localization? (e.g. `Also, reset...`)

Reply:
Thanks for calling this out! I've also noticed some copy updates to the footer in Figma, so carried those over in the latest PR update. Could you have another look? Thanks!